R/workflow.R

```diff
@@ -307,6 +321,11 @@ copy_nonmem_model_from <- function(
   new_yaml_path <- yaml_ext(.new_model)
   save_mod_yaml(.new_spec, .out_path = new_yaml_path)
 
+  # optionally open the control stream for editing
+  if (isTRUE(.open_mod_file)) {
```

Comment:
these are the types of innocent helpers that can complicate an api. For example, "oh why use that, I'd prefer to open it via the rstudio api when xyz". These types of patterns are often best handled as a separate function like edit_mod_file(). Think of this similar to how dplyr functions don't have like a .preview or something arg that will automatically open the df in the viewer. Instead, if you want that you learn to `df %>% ... %>% View()`